deploy-airflow-dags/action.yml

```diff
@@ -0,0 +1,49 @@
+name: "Deploy Airflow dags"
+description: Deploy dags to Airflow instance
+branding:
+  icon: refresh-ccw
+  color: green
+inputs:
+  project:
```

Comment:
suggestion: this is in fact the a prefix of the dag name that is being deployed, maybe we should change with a more explanatory name